packages/website/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```

Comment:
Maybe initialize the file with atti-components meta data ?

Reply:
Done